cloudformation/stack.yaml

```diff
@@ -1244,12 +1244,7 @@ Resources:
                  - ses:VerifyDomainIdentity
                  - ses:VerifyEmailIdentity
                Resource:
-                 - "*"
-             - Effect: Allow
-               Action:
-                 - logs:*
-               Resource: arn:aws:logs:*:*:*
```

Comment:
I'm always hesitant to get rid of things in the template, but certainly open to changes. What does getting rid of the logs do?

Reply:
You should be included on the email chain that explain why we need to get raid of this.  it's was something AWS told us wasn't needed and also is a potential risk for users.